README.md

```diff
@@ -903,57 +903,82 @@ Homebrew causes insecure directories like `/usr/local/share/zsh/site-functions`
-- the current version in use is `14.15.4`
-- you are using the `zsh` shell
-- you have Rosetta 2 installed (macOS prompts you to install Rosetta 2 the first time you open a Intel-only non-command-line application, or you may install Rosetta 2 from the command line with `softwareupdate --install-rosetta`)
-
-```sh
-# Check what version you're running:
-$ node --version
-v14.15.4
-# Check architecture of the `node` binary:
-$ node -p process.arch
-arm64
-# This confirms that the arch is for the M1 chip, which is causing the problems.
-# So we need to uninstall it.
-# We can't uninstall the version we are currently using, so switch to another version:
-$ nvm install v12.20.1
-# Now uninstall the version we want to replace:
-$ nvm uninstall v14.15.4
-# Launch a new zsh process under the 64-bit X86 architecture:
-$ arch -x86_64 zsh
-# Install node using nvm. This should download the precompiled x64 binary:
-$ nvm install v14.15.4
-# Now check that the architecture is correct:
-$ node -p process.arch
-x64
-# It is now safe to return to the arm64 zsh process:
-$ exit
-# We're back to a native shell:
-$ arch
-arm64
-# And the new version is now available to use:
-$ nvm use v14.15.4
-Now using node v14.15.4 (npm v6.14.10)
-```
+Experimental support for the M1 architecture was added in NodeJS v15.3 and full support
+was added in v16.0. Because of this, if you try to install older versions of Node as usual
+you will probably experience either compiliation errors when installing Node or
+out-of-memory errors while running your code.
+
+So, if you want to run a version prior to v16.0 on an M1 Mac, it may be best to compile
+Node targetting the x86_64 Intel architecture so that Rosetta 2 can translate the x86_64
+processor instructions to ARM-based Apple Silicon instructions. Here's what you will need
+to do:
+
+- Install Rosetta, if you haven't already done so
+
+  ```sh
+  $ softwareupdate --install-rosetta
+  ```
+
+  You might wonder, "how will my M1 Mac know to use Rosetta for a version of Node compiled
+  for an Intel chip?". If an executable contains only Intel instructions, macOS will
+  automatically use Rosetta to translate the instructions.
+
+- Open a shell that's running using Rosetta
+
+  ```sh
+  $ arch -x86_64 zsh
+  ```
+
+  Note: This same thing can also be accomplished by finding the Terminal or iTerm App in
+  Finder, right clicking, selecting "Get Info", and then checking the box labeled
+  "Open using Rosetta".
+
+  Note: This terminal session is now running in `zsh`. If `zsh` is not the shell you
+  typically use, `nvm` may not be `source`'d automatically like it probably is for your
+  usual shell through your dotfiles. If that's the case, make sure to source `nvm`.
+
+  ```sh
+  $ source "${NVM_DIR}/.nvm/nvm.sh
+  ```
+
+- Install whatever older version of Node you are interested in. Let's use 12.22.1 as an
+  example. This will fetch the Node source code and compile it, which will take several
+  minutes.
```

Comment:
```suggestion
- Install whatever older version of node you are interested in. Let's use 12.22.1 as an example.
  This will fetch the node source code and compile it, which will take several minutes.
```